src/steps/git.rs

```diff
@@ -292,10 +292,12 @@ impl RepoStep {
     }
 
     /// Try to pull a repo.
-    async fn pull_repo<P: AsRef<Path>>(&self, ctx: &ExecutionContext<'_>, repo: P) -> Result<()> {
+    async fn pull_repo<P: AsRef<Path>>(&self, ctx: &ExecutionContext<'_>, repo: P, verbose: bool) -> Result<()> {
```

Comment:
We don't need an extra argument here, we can access the `--verbose` option through `ctx.config().verbose()`